cluster-provision/k8s-fedora/check-cluster-up.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
IDK, maybe we can just use a symlink here instead of copying?

Reply:
I copied it is case it will be necessary to add more tests in case nodes are fedora, your call.